server/middlewares/verify_jwt.js

```diff
@@ -0,0 +1,14 @@
+require("dotenv").config();
+const { verify } = require("jsonwebtoken");
+
+export function verify_jwt(jwt) {
+    try {
+        const decoded = verify(jwt, process.env.JWT_SECRET);
+        return { payload: decoded, expired: false };
```

Comment:
Auch wenn der JWT abgelaufen ist, wird er korrekt decoded. Daher in der Payload sollte ein expire Zeitpunkt stehen, welches zusätzlich geprüft werden muss.

Reply:
https://github.com/auth0/node-jsonwebtoken#tokenexpirederror

Reply:
sollte laut der docu nen error werfen, wenn der token expired ist

Reply:
Dann hatte ich nicht auf dem Schirm das man im JWT direkt ein ExpireDate mitgeben kann.